File: VotacaoAlterData-App/src/app/_models/Recurso.ts
import { ItemRecurso } from "./ItemRecurso";
import { RecursoUser } from "./RecursoUser";


export class Recurso
{
  recursoId: string;
  dataCadastro: Date;
  itensRecurso: ItemRecurso[];
  recursosUsers: RecursoUser[];
  descricao: string;
  ativo: boolean;
}

File: VotacaoAlterData.Domain/Entity/Recurso.cs
using System;
using System.Collections.Generic;
using System.Security.Authentication.ExtendedProtection;
using System.Text;

namespace VotacaoAlterData.Domain.Entity
{
    public class Recurso
    {
        public int RecursoId { get; set; }
        public string Descricao { get; set; }
        public DateTime DataCadastro { get; set; }
        public virtual List<ItemRecurso> ItensRecurso { get; set; }
        public virtual List<RecursoUser> RecursosUsers { get; set; }
        public bool Ativo { get; set; }
    }
}
